WordPressKit/AccountSettings.swift

```diff
@@ -32,7 +33,8 @@ public struct AccountSettings {
                 webAddress: String,
                 language: String,
                 tracksOptOut: Bool,
-                blockEmailNotifications: Bool) {
+                blockEmailNotifications: Bool,
+                twoStepEnabled: Bool) {
```

Comment:
@salimbraksa I think this is a breaking change, since a new argument is added to the public initialiser? Maybe we need a major version bump for this change.